### PR TITLE
Microsoft.LAPS version 6.2.0.0

### PR DESCRIPTION
**Pull request by jezhodges**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58332)

### CODE REVIEW

**Review by jedieaston — DISMISSED** [review]
How did we miss LAPS?

File: manifests/m/Microsoft/LAPS/6.2.0.0/Microsoft.LAPS.installer.yaml [hunk 1]
```diff
@@ -3,11 +3,14 @@
 
 PackageIdentifier: Microsoft.LAPS
 PackageVersion: 6.2.0.0
+Scope: machine
 InstallerLocale: en-US
-InstallerType: wix
+InstallerType: msi
+InstallerSwitches:
+  Silent: /qn /norestart
+  SilentWithProgress: /qn /norestart
```
Comment (jedieaston) [inline manifests/m/Microsoft/LAPS/6.2.0.0/Microsoft.LAPS.installer.yaml:11]: Can you revert this? wingetcreate detected the right thing, the installer was created with WiX. winget automatically adds the correct installer switches for WiX installers.

**Review by jedieaston — CHANGES_REQUESTED** [review]

**Comment by jezhodges** [comment]
@jedieaston noticed that the installer can install just the password update service or the service + LAPS UI. Do we need to differentiate between these two options?

**Comment by jedieaston** [comment]
We can't right now in winget. Typically we just take the default and the user can use `winget install --override` to change the options themselves.

**Review by jedieaston — APPROVED** [review]

**Pull request merged**